.github/workflows/ESlint.yaml

```
@@ -0,0 +1,47 @@
+name: ESlint
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v*
+
+jobs:
+  code:
+    name: Versões NodeJS
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version:
+          - 14.x
+          - 15.x
+
+    steps:
+      - name: Obtendo versão mais nova do Codigo
+        uses: actions/checkout@v2
+
+      - name: Configurando NodeJS ${{ matrix.node-version }}
+        uses: actions/setup-node@main
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Instalar as dependências do npm
+        run: npm install
+
+      - name: Rodando ESlint
+        run: npm run fix
```

```suggestion
        run: npm run lint
```

O script `fix` executa o lint corrigindo os problemas encontrados, se executarmos ele na pipeline iremos mascarar os problemas existentes.

É preciso criar um novo script no package chamado (sugestão) `lint`, aonde vai executar o comando de `fix` sem o comando `--fix`.

Se possível renomear o `fix` para `lint:fix`.